catkit2/services/thorlabs_cube_motor_kinesis/thorlabs_cube_motor_kinesis.py

```diff
@@ -0,0 +1,233 @@
+"""
+The library calls for the Thorlabs cube motor in this script make use and adapt code from
+https://github.com/Thorlabs/Motion_Control_Examples/tree/main/Python, which is licensed under the MIT License.
+Their original copyright notice and permission notice are included below.
+
+Copyright (c) 2021 Thorlabs
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+"""
+import os
+import sys
+
+from ctypes import c_ulong, c_int, c_ushort, c_double
+from ctypes import cdll, CDLL, byref, create_string_buffer
+import numpy as np
+
+from catkit2.testbed.service import Service
+
+
+class ThorlabsCubeMotorKinesis(Service):
+    """
+    Service for controlling a Thorlabs cube motor.
+    """
+
+    def __init__(self):
+        super().__init__('thorlabs_cube_motor_kinesis')
+
+        self.cube_model = self.config['cube_model']
+        self.stage_model = self.config['stage_model']
+        self.motor_positions = self.config['positions']
+        self.motor_type = 0
+        self.serial_number = str(self.config['serial_number']).encode("UTF-8")
+
+        # Load the Thorlabs DLLs.
+        default_dll_path = r"C:\Program Files\Thorlabs\Kinesis"

```

Comment:
Wouldn't it be cleaner to use an environment variable, to set up when installing the dlls?